client/src/state/toasts/ToastsActions.js

```diff
@@ -0,0 +1,40 @@
+import ACTION_TYPES from './ToastsActionTypes';
+import { v4 as uuid } from 'uuid';
+
+const defaultOptions = {
+  stayTime: 3000,				// time in miliseconds before the item has to disappear
+  text: '',					// content of the item
+  stay: false,				// should the notice item stay or not?
+  type: 'notice' 			// could also be error, succes
+};
+
+/**
+ * Sets global config for the application.
+ *
+ * @param {object} config
+ */
+export function display(options) {
```

Comment:
I'm going to add a few alias here for displaying different message type:
- `success()`
- `warning()`
- `error()`
- `notice()`

Reply:
Would it be notice or info? 🤔

Reply:
I think BS uses **info** as a class so we should use that.

Reply:
`info()` it is.